templates/apps/sumo-aws-apps/firewall-manager-org/firewall-manager-org-waf-network-firewall-policy.yaml

```diff
@@ -271,9 +267,7 @@ Resources:
                     - KinesisFirehoseDeliveryStreamArn: !If [create_delivery_stream, !GetAtt KinesisFirehoseDeliveryStream.Arn ,''] 
                 - ''
           
-  rFmsRegionalWafWindowsPolicy:
-    DependsOn:
```

Comment:
these policies needs KinesisFirehoseDeliveryStreamArn so it should exists before we create these policies